ethpm/tools/builder.py

```diff
@@ -524,14 +527,35 @@ def _to_disk(
     else:
         disk_manifest_name = manifest["version"] + ".json"
 
-    if prettify:
-        contents = json.dumps(manifest, sort_keys=True, indent=4)
-    else:
-        contents = json.dumps(manifest, sort_keys=True, separators=(",", ":"))
+    contents = format_manifest(manifest, prettify)
 
     if (cwd / disk_manifest_name).is_file():
         raise ManifestBuildingError(
             "Manifest: {0} already exists in cwd: {1}".format(disk_manifest_name, cwd)
         )
     (cwd / disk_manifest_name).write_text(contents)
     return manifest
+
+
+def to_ipfs(
```

Comment:
The name of this is funny to me.  I'd expect it to be `pin_ipfs` or `pin_to_ipfs` as those are still reasonably concise and more explicit about behavior.

Reply:
Works for me - I chose `to_ipfs` b/c we have `to_disk` for writing the manifest to disk. Is that a reasonable name or worth changing to something like `write_to_disk`

Reply:
I'm a fan of symmetry/consistency for APIs so :+1: .  For some reason `to_disk` is intuitive for me, but `to_ipfs` wasn't

Reply:
Sorry is that 👍 to 
1 - `to_disk` and `to_ipfs` 
pr
2 - `pin_to_ipfs` and `write_to_disk`

Reply:
2 (I still think `to_ipfs` is non-intuitive)